### PR TITLE
Create OpenSource : 3. Blog what you do and be more social

### LINKED ISSUE #32: OpenSource : 3. Blog what you do and be more social
### Description :

Blogging is a very good approach to build an audience for your open source projects. `Introducing your projects` and writing `how-to-type blog posts` would be nice for **your future community.**

### Note : 

- Changes should be made inside the `Open_Source/` directory & `Opensource` branch.
-  Issue only for `GWOC'21` contributors.
-  Issue will be assigned on a **first come first serve basis**, **1 Issue  ==  1 PR.**  

## Happy Contributing 🙌

### PR DESCRIPTION
**Pull request by Ash-KODES**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #32

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [ ] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [x] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Aryamanz29** [comment]
@Ash-KODES  Some changes: 👇
- Add extension `.md` to your file
- Add` images, flowcharts, graphs to explain what you did
Once you are done with the above changes, Let me know 😉

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]
@Ash-KODES  Some changes: 👇
- Add extension `.md` to your file
- Add `images, flowcharts, graphs` to explain topic
-  Remove` extra lines` from your doc
Once you are done with the above changes, Let me know 😉

**Comment by Ash-KODES** [comment]
> @Ash-KODES Some changes: 👇
> 
> * Add extension `.md` to your file
> * Add `images, flowcharts, graphs` to explain what you did
>   Once you are done with the above changes, Let me know 😉

ok

**Comment by Aryamanz29** [comment]
@Ash-KODES Any updates ??

**Comment by Ash-KODES** [comment]
> @Ash-KODES Any updates ??

ya i am on it!

File: Open_Source/OpenSource : 3. Blog what you do and be more social [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# **WHAT IS BLOGGING ?**
+A Blog is an online journal where people write about things that matters to them. A lot of people use it as a diary and a lot of people who blog to make a profit out of it.
+
+Blogging refers to writing, photography, and other media that's self-published online. Blogging started as an opportunity for individuals to write diary-style entries, but it has since been incorporated into websites for many businesses. The hallmarks of blogging include frequent updates, informal language, and opportunities for readers to engage and start a conversation.
+
+
+---
+
+
```
Comment (Aryamanz29) [inline Open_Source/OpenSource : 3. Blog what you do and be more social:9]: Remove extra lines from `doc`

File: Open_Source/OpenSource : 3. Blog what you do and be more social [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+**5. Write the main content**
+
+Now you need to write the content of your blogpost and the outline of your project,and make sure you keep updating it with the project devlopment.
+
+
+
+---
+
+
```
Comment (Aryamanz29) [inline Open_Source/OpenSource : 3. Blog what you do and be more social:79]: Remove extra lines

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

**Comment by Ash-KODES** [comment]
@Aryamanz29 I have made a new pr please review it

**Comment by Aryamanz29** [comment]
Closing this PR , Since same one already been  merged

**Pull request closed without merging**